pay-api/src/pay_api/services/gcp_queue/gcp_queue.py

```diff
@@ -0,0 +1,184 @@
+# Copyright © 2023 Province of British Columbia
```

Comment:
Ideally we need a note stating this is going to be replaced, if we're loading it from LEAR's library?